test/OrchardCore.Commerce.Tests.UI/Tests/LocalizedProductTests/LocalizedProductBehaviourTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using OrchardCore.Commerce.MoneyDataType;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OrchardCore.Commerce.Tests.UI.Tests.LocalizedProductTests;
+
+public class LocalizedProductBehaviourTests : UITestBase
+{
+    private const string LocalizedTitle = "Honosított Termék"; // #spell-check-ignore-line
+    private const string LocalizationsButtonPath =
+        "//li[contains(@class, 'list-group-item') and .//a[contains(., 'Test Localized Product')]]//div[@title = 'Localizations']//button";
```

Comment:
This is only used once. But if you think it might be used in the future, you can also store the whole `By.XPath`:

```suggestion
    private static readonly By LocalizationsButtonPath =
        By.XPath("//li[contains(@class, 'list-group-item') and .//a[contains(., 'Test Localized Product')]]//div[@title = 'Localizations']//button");
```